Makefile

```diff
@@ -652,7 +654,13 @@ verify-tiltfile: ## Verify Tiltfile format
 
 .PHONY: verify-container-images
 verify-container-images: ## Verify container images
-	TRACE=$(TRACE) ./hack/verify-container-images.sh
+	TRACE=$(TRACE) ./hack/verify-container-images.sh $(TRIVY_VER)
+
+.PHONY: verify-licenses
+verify-licenses: ## Verify licenses
+	TRACE=$(TRACE) ./hack/verify-licenses.sh $(TRIVY_VER)
```

Comment:
There is also a github action if we wanted to go that path https://github.com/aquasecurity/trivy-action

Reply:
Good to know - for now though I like the pattern of running all verify jobs at once. And I think there's a general community preference for putting stuff on prow when that's an option.

Reply:
I have considered the trivy action, but running it for every image we were producing wasn't really efficient while now we are pulling trivy + its database only once and reusing it for all the images
But some time is passed, so might be worth re-evaluating

Reply:
E.g. stuff like k/k seems to ignore go.sum files, they are not accepting MPL licences, ...